static/bidder-params/mobfoxpb.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Mobfox Adapter Params",
+  "description": "A schema which validates params accepted by the Mobfox adapter",
+  "type": "object",
+  "properties": {
+    "TagID": {
```

Comment:
Are you sure you want to use `TagID` instead of `tagID` or `tagid`? It's common in JavaScript/JSON to start with a lowercase letter and it's common in OpenRTB to use all lowercase letters. Either is fine in my view, as I see this is decision owned by the adapter.

Just be aware that changes to this after merging are disastrous to publishers and should be avoided.

Reply:
This made to show that  this property and property from bidder extension are the same, we have no experienced golang developer, so this brings little better understanding. Let's left it untouched

Reply:
The name here doesn't have to match the field name on the programming language. If you'd like to keep it TagID, then sure. Would you want to name it `placementId` to match with your existing PBJS adapter? A lot publishers prefer to have the same variables on both PBS and PBJS so they can easily retarget, and using different parameter names messes with that a bit.

Reply:
Let's keep it with current name